src/mscorlib/src/System/Text/Encoding.cs

```diff
@@ -978,10 +1003,44 @@ public virtual byte[] GetBytes(String s)
             return bytes;
         }
 
+        // Returns a byte array containing the encoded representation of the given
+        // string range.
+        //
+        [Pure]
+        public unsafe virtual byte[] GetBytes(string s, int index, int count)
```

Comment:
Since there aren't pointer types in the method signature the unsafe could be moved from the method signature to a block around the `fixed` statement.  Toe-may-toe/toe-mah-toe, really.